src/index.ts

```diff
@@ -70,6 +83,11 @@ export async function init(config: IClientConfig): Promise<IEppoClient> {
   );
   clientInstance = new EppoClient(configurationRequestor, poller);
   clientInstance.setLogger(config.assignmentLogger);
+
+  clientInstance.setAssignmentCache(
+    config.assignmentCache ?? new LRUAssignmentCache(ASSIGNMENT_CACHE_DEFAULT_MAX_ENTRIES),
```

Comment:
I want people to be able to disable the cache if they want to - looking at this more carefully I believe an incoming value of `undefined` would be ignored, so I do need to re-implement this.

Will also add a test to make sure an `undefined` value for `config.assignmentCache` yields no cache